lib/api/src/rest/schema.rs

```diff
@@ -0,0 +1,87 @@
+use std::collections::HashMap;
+
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+/// Type for dense vector
+pub type DenseVector = Vec<segment::data_types::vectors::VectorElementType>;
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize, JsonSchema)]
```

Comment:
Can we remove right away the `JsonSchema` attribute on the original segment types?

Reply:
Yep! Forgot to do that

Reply:
For this type, for others did it )

Reply:
It seems I did it for all moved types: `Vector`, `VectorStruct` and `ScoredPoint`